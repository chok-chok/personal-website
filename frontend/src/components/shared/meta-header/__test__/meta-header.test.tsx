import * as React from 'react';
import MetaHeader, { MetaHeaderProps } from '../meta-header';
import { shallow } from 'enzyme';

const props: MetaHeaderProps = {
  pageTitle: 'This is my homepage',
  metaDescription: 'Welcome to my website',
  indexStatus: true,
  urlPath: 'https://www.yunjaeoh.com',
};

describe('Test Header component', () => {
  it('should renders MetaHeader', () => {
    const tree = shallow(<MetaHeader {...props} />);
    expect(tree).toMatchSnapshot();
  });
});
