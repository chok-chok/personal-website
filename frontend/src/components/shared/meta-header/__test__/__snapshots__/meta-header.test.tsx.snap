// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test Header component should renders MetaHeader 1`] = `
<Fragment>
  <Head>
    <meta
      charSet="utf-8"
    />
    <meta
      content="IE=edge, chrome=1"
      httpEquiv="X-UA-Compatible"
    />
    <meta
      content="width=device-width, initial-scale=1, maximum-scale=1, minimum-scale=1"
      name="viewport"
    />
    <link
      href="https://cdnjs.cloudflare.com/ajax/libs/normalize/8.0.0/normalize.min.css"
      rel="stylesheet"
    />
    <title>
      This is my homepage
    </title>
    <meta
      content="Welcome to my website"
      name="description"
    />
    <meta
      content="index follow"
      name="robots"
    />
    <meta
      content="https://www.yunjaeoh.com"
      property="og:url"
    />
    <meta
      content="website"
      property="og:type"
    />
    <meta
      content="This is my homepage"
      property="og:title"
    />
    <meta
      content="Welcome to my website"
      property="og:description"
    />
    <meta
      content=""
      property="og:image"
    />
  </Head>
</Fragment>
`;
